themes/wporg-support/template-parts/content-single.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Template part for displaying page content in page.php.
+ *
+ * @link https://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package WPBBP
+ */
+
+?>
+
+<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+	<?php if ( is_page_template( 'page-forums-sidebar.php' ) ) : ?>
+		<?php bbp_breadcrumb(); ?>
+	<?php endif; ?>
+
+	<header class="entry-header">
+		<?php the_title( '<h1 class="entry-title">', '</h1>' ); ?>
+	</header><!-- .entry-header -->
+
+	<div class="entry-content">
+		<div class="container">
+			<?php
+			the_content();
+
+			wp_link_pages( array(
+				'before' => '<div class="page-links">' . esc_html__( 'Pages:', 'wporg-forums' ),
+				'after'  => '</div>',
+			) );
+			?>
+		</div>
+	</div><!-- .entry-content -->
+
+	<footer class="entry-footer">
+		<?php
+		edit_post_link(
+			sprintf(
+				/* translators: %s: Name of current post */
+				esc_html__( 'Edit %s', 'wporg-forums' ),
```

Comment:
`_x()` maybe?

Reply:
All these are from the existing theme, I want to maintain as much of it as possible to speed up the next commit step.